src/table/TableBodyWrapper.jsx

```diff
@@ -35,16 +35,16 @@ const useStyles = makeStyles({
   },
 });
 
-const TableBodyWrapper = ({
```

Comment:
@a-m-dev hi, I am curious that any specific reason to change to use `function` ?

Reply:
Hey Li, 
Not that much, i guess, i might have change it because of `sinon.stub` thing that we had a bit of problem in our test files, but both are fine